apps/ios/SimpleX SE/ShareModel.swift

```diff
@@ -215,17 +208,18 @@ class ShareModel: ObservableObject {
                 }
             case let .sndFileCompleteXFTP(_, ci, _):
                 guard isMessage(for: ci) else { continue }
-                    if isGroupChat {
-                        await MainActor.run { bottomBar = .loadingSpinner }
-                    }
-                    await ch.completeFile()
-                    if await !ch.isRunning { break }
+                if isGroupChat {
+                    await MainActor.run { bottomBar = .loadingSpinner }
+                }
+                await ch.completeFile()
+                if await !ch.isRunning { break }
             case let .chatItemStatusUpdated(_, ci):
                 guard isMessage(for: ci) else { continue }
                 if let (title, message) = ci.chatItem.meta.itemStatus.statusInfo {
+                    // `title` and `message` already localized and interpolated
                     return ErrorAlert(
-                        title: "\(title)",
-                        message: "\(message)"
+                        title: LocalizedStringKey(title),
+                        message: LocalizedStringKey(message)
```

Comment:
why this change?

https://developer.apple.com/documentation/swiftui/localizedstringkey/init(_:)

This says that the parameter will be used as a localisation key, which is not what we want.